mediasoup-client/src/androidTest/jni/generated_mediasoupclient_jni/jni/Parameters_jni.h

```diff
@@ -0,0 +1,116 @@
+// Copyright 2014 The Chromium Authors. All rights reserved.
```

Comment:
why add this file in androidTest? seems no need.

Reply:
Looks like Parameters jni stuf is only used in androidTest : in [Parameters.java](https://github.com/haiyangwu/mediasoup-client-android/blob/dev/mediasoup-client/src/androidTest/java/org/mediasoup/droid/data/Parameters.java) the AndroidTest code

Is it planned to be migrated to the main code ?